features/step_definitions/admin_statistical_data_sets_steps.rb

```diff
@@ -3,6 +3,6 @@
 
   data_sets = @new_edition.statistical_data_sets
 
-  assert_equal 1, data_sets.count
-  assert_equal data_set_name, data_sets.first.title
+  expect(1).to eq(data_sets.count)
```

Comment:
A really minor point, but I wonder if this would read better the other way round (e.g. `expect(data_sets.count).to eq(1)`)?

Reply:
Ah, good spot! Have switched the args around.